debian/control

```diff
@@ -0,0 +1,26 @@
+Source: cligen
+Section: devel
+Priority: optional
+Maintainer: roma55592@yandex.ru
+Build-Depends:
+ debhelper-compat (= 12),
```

Comment:
Can debhelper-compat (= 13) be used?

Reply:
@YotaYota I do not remember which version it is installed in ubuntu 22.04. I wanted it to be compatible with debian 11 since it is popular, I will try now, but I think not

Reply:
@YotaYota yes, 13 no in ubuntu20.04